sdcm/sct_events.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import zmq
+import json
+import errno
+import os
+import signal
+
+from multiprocessing import Process, Value, Event
+
+LOGGER = logging.getLogger(__name__)
+
+
+class EventsDevice(Process):
```

Comment:
Look in https://github.com/zeromq/pyzmq/blob/master/zmq/devices/basedevice.py
I think it has been already implemented there

Reply:
I'll later look, on how to integrate it, seems like it's almost what I'm doing here